CMakeLists.txt

```diff
@@ -354,45 +367,55 @@ if(ENABLE_CPPROFILER)
 endif()
 
 find_package(Threads)
-target_link_libraries(gecodesupport ${CMAKE_THREAD_LIBS_INIT})
-target_link_libraries(gecodekernel gecodesupport)
-target_link_libraries(gecodesearch gecodekernel)
-target_link_libraries(gecodeint gecodekernel)
-target_link_libraries(gecodeset gecodeint)
-target_link_libraries(gecodefloat gecodeint)
-target_link_libraries(gecodeminimodel gecodeint gecodeset gecodesearch)
-target_link_libraries(gecodedriver gecodeint)
-target_link_libraries(gecodeflatzinc gecodeminimodel gecodedriver)
+target_link_libraries(support ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Why rename without gecode in the name? That said, might make sense to add gecode in the function… and also set up alias targets…

Reply:
The removal of "gecode" from the exported targets during installation, with the introduction of the gecode:: namespace, aims to prevent naming conflicts and provides a clean approach

Reply:
But that means that the actual libraries will also have the names here and those are very common names that are likely to conflict with other things. Right?

Reply:
agree, but I still think we should have a cleaner target name. If you're on board with this idea, i  can make an improvement by adding the following set_target_properties .
```
set_target_properties(targetdummy PROPERTIES
    OUTPUT_NAME gecode_targetdummy 
)